lib/ProductOpener/Tags.pm

```diff
@@ -195,6 +195,8 @@ use MIME::Base64 qw(encode_base64);
 use POSIX qw(strftime);
 use LWP::UserAgent ();
 use Encode;
+use IO::Compress::Gzip qw(gzip $GzipError);
```

Comment:
Are these standard modules or should we pin a version in `cpanfile`?

Reply:
I noticed that both `IO` modules are transitive dependencies of `LWP::UserAgent` and `HTTP::Response`. Probably not required to list them in cpanfile.

Reply:
They were both modules we were using already so assumed no further changes were needed